app/index.js

```diff
@@ -103,11 +76,18 @@ var onError = function(error){
   }
   process.exit(-1);
 };
-var onListening = function(){
-  var listenurl = (nconf.get('https')?'https':'http:')+'://'+nconf.get('listen')+':'+nconf.get('port');
+function onListening() {
+  let listenurl = (nconf.get('https')?'https':'http:')+'://'+nconf.get('listen')+':'+nconf.get('port');
```

Comment:
could we refactor these nconf.get calls to the top of the file, seems like they are called quite a few times. I assume we are not expecting them to change. Possible names:
* https -> {boolean} const usingHttps
* listen -> const address 
* port -> const port